app/controllers/v3/service_instances_controller.rb

```diff
@@ -199,6 +199,8 @@ def credentials
     service_instance_not_found! unless service_instance && can_read_service_instance?(service_instance)
     unauthorized! unless permission_queryer.can_read_secrets_in_space?(service_instance.space_id, service_instance.space.organization_id)
 
+    service_event_repository.record_user_provided_service_instance_event(:show, service_instance)
```

Comment:
It took us a moment to figure out why there was only a `audit.user_provided_service_instance.show` event here and not also a `audit.service_instance.show`.  That is due to the way we have set up the endpoint such that there is only user provided service instance, so this seems good as well